firestore/swift/firestore-smoketest/ViewController.swift

```diff
@@ -673,6 +674,18 @@ class ViewController: UIViewController {
         // [END listen_document_local]
     }
 
+    private func listenWithMetadata() {
+      // [START listen_with_metadata]
+      // Listen to document metadata.
+      let options = DocumentListenOptions().includeMetadataChanges(true);
+
+      db.collection("cities").document("SF")
+        .addSnapshotListener(options:options, listener: { documentSnapshot, error in
+          // ...
+      })
```

Comment:
bonus points for trailing closure syntax (also space between colon and argument in Swift)

```swift
db.collection("cities").document("SF").addSnapshotListener(options: options) { documentSnapshot, error in
  // ...
}
```